packages/components/src/InputFile/InputFile.test.tsx

```diff
@@ -388,7 +396,11 @@ describe("Content", () => {
       />,
     );
 
-    const button = screen.getByRole("button", { name: "Upload file" });
+    const button = screen.getByRole("button", {
```

Comment:
This is because the dropzone wrapper also gets the button role so this is to find the actual button and make sure it is styled correctly